vllm/entrypoints/api_server.py

```diff
@@ -107,6 +110,13 @@ async def stream_results() -> AsyncGenerator[bytes, None]:
         engine_args, usage_context=UsageContext.API_SERVER)
 
     app.root_path = args.root_path
+
+    logger.info("Available endpoints are:")
```

Comment:
Maybe more align to the terms we are using.
```suggestion
    logger.info("Available routes are:")
```